adage/analyze/tests.py

```diff
@@ -625,19 +642,21 @@ def test_activity_mlmodel_filter(self):
         """
         Test "activity/?sample=<id>&mlmodel=<ml_id>" API.
         """
-        uri = (self.baseURI + "activity/?sample=" +
-               str(self.random_object(Sample).id) + "&mlmodel=" +
-               str(self.random_object(MLModel).id))
+        uri = self.baseURI + "activity/"
+        data = {
```

Comment:
This is the actual bug fix. According to the (documentation)[http://django-tastypie.readthedocs.io/en/latest/testing.html#get], we should pass all parameters this way, even for our `GET` endpoints.